PowerPointLabs/PowerPointLabs/SyncLab/SyncLabShapeStorage.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+
+using Microsoft.Office.Interop.PowerPoint;
+
+using PowerPointLabs.Models;
+
+namespace PowerPointLabs.SyncLab
+{
+    public class SyncLabShapeStorage : PowerPointPresentation
+    {
+#pragma warning disable 0618
```

Comment:
The `warning disable 0618` is to get the clipboard's content. We considered using action framework but it can only be applied to ribbon objects, but not the panel, so we had to resort to this.